CHANGELOG.latest.md

```diff
@@ -0,0 +1,7 @@
+## 3.9.0
+- Added support for StoreKit Config Files and StoreKitTest testing
+    https://github.com/RevenueCat/purchases-ios/pull/407
+- limit running integration tests to tags and release branches
+    https://github.com/RevenueCat/purchases-ios/pull/406
+- added deployment checks
+    https://github.com/RevenueCat/purchases-ios/pull/404
```

Comment:
I believe there should be a line break at the end

Reply:
Nitpick. I also think we shouldn't include the version here and fastlane should add it automatically when appending this to the `CHANGELOG.md`. That way we don't have the version number twice in the releases page.

![Screen Shot 2020-12-07 at 8 19 06 AM](https://user-images.githubusercontent.com/664544/101376017-ebddd180-3864-11eb-88c2-5761a8bc9426.png)

Do you think that's easy to do?


Apart from that, this is cool, because we can also add stuff to the `CHANGELOG.latest.md` when doing a PR.

Reply:
there's a line break, but GH doesn't show it for whatever reason. 
Great call about removing the version number, adding that now

Reply:
@vegaro done